website/docs/docs/dbt-cloud/cloud-api.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "dbt Cloud API"
+id: "cloud-api"
+---
+
+### Overview
+
+If you are on the `Team` or `Enterprise` dbt Cloud plan, you will have access to the 
+dbt Cloud API.
+
+Common uses for our API would be:
+
+- Downloading artifacts after a job has completed 
+- Kicking off a job run 
+- Creating/modifying/deleting objects in dbt Cloud 
```

Comment:
```suggestion
The dbt Cloud API can be used to:

- Download artifacts after a job has completed 
- Kick off a job run from an orchestration tool
- Manage your dbt Cloud account
```

Reply:
huh - I see this one market as resolved, but the change wasn't actually applied. My thinking here is just that we shouldn't call this "our" API, we should call it "the" API :)